driver/network_driver.go

```diff
@@ -76,6 +78,11 @@ func NewNetworkDriver(client *datastoreClient.Client) network.Driver {
 		labelEndpoints: strings.EqualFold(os.Getenv(LABEL_ENDPOINTS_ENVKEY), "true"),
 	}
 
+	ns := os.Getenv(NAMESPACE_ENVKEY)
```

Comment:
Won't this configure the namespace for all users of this driver? I don't think that's right - I don't think we want a per-node namespace configuration option.

Reply:
Could we use a container label instead?

Reply:
oh! I misunderstand you! However use container label will query docker in create/join/delete, I think that's will make plugin require docker's response. When docker daemon restarting, it will cause block problems. I think ```libnetwork.<network_name>``` will be better, I will change it on Monday.